admin/src/scenes/volontaires/view/phase2bis/applicationList2.js

```diff
@@ -121,10 +153,37 @@ const Hit = ({ hit, index, young, onChangeApplication }) => {
 
           {/* places disponibles */}
           <div className="flex basis-[25%]">
-            {mission.placesLeft <= 1 ? (
-              <div className="font-medium text-xs text-gray-700 "> {mission.placesLeft} place disponible</div>
+            {["VALIDATED", "IN_PROGRESS", "DONE"].includes(hit.status) ? (
+              <div className="flex flex-col">
+                {contract?.invitationSent === "true" ? (
+                  <div className="font-medium text-xs text-gray-700 ">Contrat {checkStatusContract(contract) === "VALIDATED" ? "signé" : "envoyé"}</div>
+                ) : (
+                  <div className="flex flex-row items-center">
+                    <div className="w-[8px] h-[8px] rounded-full bg-orange-500" />
+                    <div className="font-medium text-xs text-gray-700 ml-1">Contrat en brouillon</div>
+                  </div>
+                )}
+                {numberOfFiles === 1 && (
+                  <div className="flex flex-row items-center mt-1">
+                    <div className="w-[8px] h-[8px] rounded-full bg-orange-500" />
+                    <div className="font-medium text-xs text-gray-700 ml-1">1 pièce jointe</div>
+                  </div>
+                )}
+                {numberOfFiles > 1 && (
+                  <div className="flex flex-row items-center mt-1">
+                    <div className="w-[8px] h-[8px] rounded-full bg-orange-500" />
+                    <div className="font-medium text-xs text-gray-700 ml-1">{numberOfFiles} pièces jointes</div>
+                  </div>
```

Comment:
si ca peut reduire la quantité de code
```suggestion
                {numberOfFiles >= 1 && (
                  <div className="flex flex-row items-center mt-1">
                    <div className="w-[8px] h-[8px] rounded-full bg-orange-500" />
                    <div className="font-medium text-xs text-gray-700 ml-1">{numberOfFiles} pièce{numberOfFiles > 1 ? "s" : ""} jointe{numberOfFiles > 1 ? "s" : ""}</div>
                  </div>
```